app/allocation/views/confirmation.njk

```diff
@@ -28,15 +29,9 @@
       </p>
 
       <p>
-        <a href="{{ ALLOCATIONS_URL }}">
+        <a href="/allocations">
           {{ t("allocations::confirmation.view_all") }}
         </a>
-        <span> {{t("or")}} </span>
```

Comment:
Maybe tidy up the removal of this translation too. Don't think it's used anywhere else.